scripts/dev/run-test/fm-run-test

```diff
@@ -25,9 +28,14 @@ on_error() {
 }
 
 on_exit() {
-    rm -Rf "$CARGO_BUILD_TARGET_DIR"
```

Comment:
This actually never ran, because nothing called `on_exit`. And that's lucky, as it breaks everything. :D